src/test/java/model/CrewsTest.java

```diff
@@ -0,0 +1,33 @@
+package model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import attendance.model.Crew;
+import attendance.model.Crews;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class CrewsTest {
+
+    @Test
+    void 등록되지_않은_닉네임으로_찾으면_예외가_발생한다() {
+        Crews crews = new Crews();
+        assertThatThrownBy(() -> crews.findCrew("빙티"))
+                .isInstanceOf(IllegalArgumentException.class);
```

Comment:
exception 테스트를 할 때에는 메시지도 꼭 함께 테스트해주세요!

```suggestion
        assertThatThrownBy(() -> crews.findCrew("빙티"))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessageContaining("등록되지 않은 닉네임입니다.");
```

Reply:
모두 수정했습니다! 메시지를 함께 테스트하지 않는다면 예상과는 다른 예외로 IllegalArgumentException이 터져도 모를 수 있기 때문에 검증해야 한다는 점에서 필요하군요